test/cri-containerd/main.go

```diff
@@ -246,3 +250,30 @@ func findTestNvidiaGPUDevice() (string, error) {
 	}
 	return strings.TrimSpace(string(out)), nil
 }
+
+// Generates a gmsa credential spec and places it at 'path'. Dir has to exist, it will not be created by
+// New-CredentialSpec.
+func generateCredSpec(path, account string) error {
+	cmd := fmt.Sprintf(`New-CredentialSpec -AccountName %s -Path "%s"`, account, path)
```

Comment:
You should be able to pass these as individual args instead of one whole string. Then you don't need to worry about escaping.